issuance/issuer.go

```diff
@@ -152,7 +152,18 @@ func LoadChain(certFiles []string) ([]*Certificate, error) {
 
 // IssuerConfig describes the constraints on and URLs used by a single issuer.
 type IssuerConfig struct {
-	UseForRSALeaves   bool
+	// Active determines if the issuer can be used to sign precertificates. All
+	// issuers, regardless of this field, can be used to sign final certificates
+	// (for which an issuance token is presented), OCSP responses, and CRLs.
+	Active bool
+
+	// Deprecated: use Active instead. All Active issuers with RSA keys can be
+	// used for RSA leaves. No issuers with ECDSA keys can be used for RSA leaves.
+	UseForRSALeaves bool
+	// Deprecated: use Active instead. If features.ECDSAForAll is false, all
```

Comment:
For feature flags, we've tended to call them "deprecated" once they've been flipped in prod and we've deleted the old functionality in Boulder; in other words, "deprecated" has been synonymous with "no-op" for feature flags. I'd like to follow that same pattern here, and not call these deprecated. Instead let's document what they do, how they interact with Active and ECDSAForAll, and note at the end that using Active is now preferred.

Reply:
(See top-level comment about being synonyms for "active".) These flags now exist solely for backwards-compatibility with old configs, and create no unique behaviors: once config parsing is complete, they no longer exist. That's why I chose the phrasing "Deprecated: use Active instead", because doing so has the exact same effect on behavior.

Reply:
That's fair enough. Since this PR changes the meaning of UseFor{ECDSA,RSA}Leaves such that these fields no longer do what they say, I think we need to document that explicitly. E.g.:

// Deprecated: synonym for Active
//
// Note: Despite the name, setting this field to true cannot add an issuer to a pool different than its key type. An active issuer will always be part of a pool based on its key type.

I would actually omit the discussion of ECDSAForAll in these comments, in favor of the text I proposed on Active.

Reply:
Good idea, done. I think "Deprecated:" has to be the last paragraph of the doc-comment for godoc to pick it up correctly, so slightly rearranged.